### ISSUE TITLE
Checking source freshness on Postgres errors out during compile

### ISSUE THREAD

**Issue by joshpeng** [fishtown-analytics/dbt#1523]
### Issue description
When I have a source setup for Postgres, if I try to run the `snapshot-freshness` tests on it, it errors out at compilation time.

`dbt source snapshot-freshness --select pg_teal --target dev-postgres`

If I delete the `{% macro postgres__datediff(first_date, second_date, datepart) %}` block in `dbt_utils` the source test proceeds.

### Results
```
Running with dbt=0.13.1
Encountered an error:
Compilation Error
  macro datediff not implemented for this adapter
  
  > in macro postgres__datediff (macros/cross_db_utils/datediff.sql)
  > called by macro adapter_macro (macros/adapters/common.sql)
  > called by macro datediff (macros/cross_db_utils/datediff.sql)
  > called by <Unknown>
```

### System information
The output of `dbt --version`:
```
installed version: 0.13.1
   latest version: 0.13.1

Up to date!
```

OS: `MacOS Mojave`
Python: `3.7.3`

**Comment by drewbanin** [comment]
Thanks for the report @joshpeng!

**Comment by drewbanin** [comment]
Hey @joshpeng - are you able to reproduce this one with dbt v0.14.0? I was hoping to get this fixed for 0.14.1, but I'm currently unable to reproduce it with either v0.13.1 or v0.14.0!

If it's still a problem for you, can you tell me:
1. which version of `dbt-utils` you're using
2. if you have any instances of calling the `datediff` in your dbt project

I initially assumed that there was some sort of `datediff` macro name collision wherein dbt used the version of `datediff` from `dbt-utils` instead of dbt's "global project", but it doesn't look like such a macro exists. I'm pretty confused where this one is coming from, and any additional info you can provide would be greatly appreciated!

**Comment by joshpeng** [comment]
Hey @drewbanin, I just tried on v0.14.0 and it doesn't seem to be an issue anymore. Please consider this closed. Thanks for looking!
